src/libs/actions/BankAccounts.js

```diff
@@ -363,6 +363,15 @@ function openWorkspaceView() {
     API.read('OpenWorkspaceView');
 }
 
+/**
+ * Set the reimbursement account loading so that it happens right away, instead of when the API command is processed.
```

Comment:
NAB

```suggestion
 * Set the reimbursement account to loading so that it happens right away, instead of when the API command is processed.
```

Reply:
In the interest of getting this merged I'm going to pass on that change for now.